providers/cstor-operator/test.yml

```
@@ -0,0 +1,223 @@
+              args:
+                executable: /bin/bash
+
+            - block:
+
+                - name: Applying cspc rbac
+                  shell: "kubectl  apply -f {{ cspc_rbac }}"
+                  args:
+                    executable: /bin/bash
+
+                - name: Applying cspc crd
+                  shell: "kubectl  apply -f {{ cspc_crd }}"
+                  args:
+                    executable: /bin/bash
+              
+              when: "'OK' not in release.msg"
+
+            - name: Checking OpenEBS-CSPC-Operator is running
+              shell: >
+                kubectl get pods -n {{ namespace }}
+                -o jsonpath='{.items[?(@.metadata.labels.name=="cspc-operator")].status.phase}'
+              register: cspc_status
+              until: "'Running' in cspc_status.stdout"
+              delay: 5
+              retries: 120
+
+            - name: Checking OpenEBS-CVC-Operator is running
+              shell: >
+                kubectl get pods -n {{ namespace }}
+                -o jsonpath='{.items[?(@.metadata.labels.name=="cvc-operator")].status.phase}'
+              register: cvc_status
+              until: "'Running' in cvc_status.stdout"
+              delay: 5
+              retries: 120
+
+            - name: Checking OpenEBS-CVC-Operator is running
+              shell: >
+                kubectl get pods -n {{ namespace }}
+                -o jsonpath='{.items[?(@.metadata.labels.app=="cstor-admission-webhook")].status.phase}'
+              register: admission_status
+              until: "'Running' in admission_status.stdout"
+              delay: 5
+              retries: 120                       
+
+          when: lookup('env','ACTION') == "provision"
+
+        - block:
+              - name: Cleaning cspc operator
+                shell: "kubectl  delete -f {{ cspc_operator_link }}"
+                args:
+                  executable: /bin/bash
+                ignore_errors: True

```

why do we need ignore_errors  here?

---

If we deleted some of the components already we are getting the error operator  so only used ignore errors @gprasath